src/devices_models/device_constructors/HydroDispatchCascade_constructor.jl

```diff
@@ -0,0 +1,98 @@
+
+"""
+Construct model for HydroGen with RunOfRiver Dispatch Formulation
+"""
+function PSI.construct_device!(
+    psi_container::PSI.PSIContainer,
+    sys::PSY.System,
+    model::PSI.DeviceModel{H, HydroDispatchRunOfRiverCascade},
+    ::Type{S},
+) where {
+    H <: HydroCascade,
+    S <: PM.AbstractPowerModel,
+}
+    devices = PSI.get_available_components(H, sys)
+
+    if !PSI.validate_available_devices(H, devices)
+        return
+    end
+
+    #Variables
+    PSI.add_variables!(psi_container, PSI.ActivePowerVariable, devices)
+    PSI.add_variables!(psi_container, PSI.ReactivePowerVariable, devices)
+    PSI.add_variables!(psi_container, PSI.SpillageVariable, devices)
+
+
+    #Constraints
+    PSI.add_constraints!(
```

Comment:
I see your previous comment on HydroReservoirCascade_constructor.jl. Does this mean we can remove these constraints as well?